### PR TITLE
Move to reusable workflows from github-actions

### PR DESCRIPTION
**Pull request by viccuad**
Depends on kubewarden/github-actions#15.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
+ifndef POLICY_TYPE
+	@printf "Please assign POLICY_TYPE to either rego or opa\n"
+	exit 1
+endif
+ifeq ($(POLICY_TYPE), rego)
+	opa build -t wasm -e policy/violation -o bundle.tar.gz policy.rego
+endif
+ifeq ($(POLICY_TYPE), opa)
 	opa build -t wasm -e policy/main utility/policy.rego -o bundle.tar.gz policy.rego
+endif
```
Comment (flavio) [inline Makefile:16]: would it be possible to print an error if `POLICY_TYPE` is neither `opa` nor `rego`?

File: Makefile [hunk 2]
```diff
@@ -1,16 +1,34 @@
...
 	tar xvf bundle.tar.gz /policy.wasm
 	rm bundle.tar.gz
+	touch policy.wasm # opa creates the bundle with unix epoch timestamp, fix it
```
Comment (flavio) [inline Makefile:19]: nice touch

**Review by flavio — APPROVED** [review]
Great :clap: 

I just left a minor comment

**Comment by viccuad** [comment]
Force pushed with added check on Makefile to notice wrong policy type, and consuming new workflow names, without `input.policy-type`.

File: Makefile (hunk 1 above)
Comment (viccuad) [reply Makefile:14]: Done, force pushed.

**Review by flavio — APPROVED** [review]
LGTM

**Review by jvanz — APPROVED** [review]

**Pull request merged**